src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs

```diff
@@ -303,24 +305,6 @@ internal ElasticPoolsOperations(SqlManagementClient client)
                     throw new Microsoft.Rest.SerializationException("Unable to deserialize the response.", _responseContent, ex);
                 }
             }
-            // Deserialize Response
```

Comment:
Swagger still documents 202, but no longer has a response body, which I believe is why this section is removed.